RAMP/ramp.py

```diff
@@ -96,8 +97,12 @@ def unpack(bundle):
 @click.option('--dependencies', callback=jsons_str_tuple_to_jsons_tuple, help='list of module dependencies: <name, uri, sha256>')
 @click.option('--output', '-o', default='module.zip', help='output file name')
 @click.option('--print-filename-only', '-P', is_flag=True, default=False, help="Print package path, but don't generate file")
+@click.option('--packname-file', default=None, help="Print package name to the file")
 @click.option('--verbose', '-v', is_flag=True, default=False, help='verbose mode: print the resulting metadata')
+@click.option('--debug', is_flag=True, default=False, help='Print interaction with Redis. Implies --verbose.')
 def pack(module, *args, **kwargs):
+    config.debug = kwargs['debug']
+    config.verbose = kwargs['verbose'] or config.debug
```

Comment:
Why do you need verbose and debug?

Reply:
`--verbose` outputs the metadata. I didn't want to break this with the the Redis output in case someone is relying on this.